ubuntu20.04/ubuntu.Dockerfile

```diff
@@ -180,6 +181,8 @@ COPY ${PACKAGE_BASE_NAME}-platform-sdk-android-ndk \
      ${PACKAGE_BASE_NAME}-platform-sdk-yams \
      ${PACKAGE_BASE_NAME}-platform-sdk-z3-cross \
      ${PACKAGE_BASE_NAME}-platform-sdk-zlib-cross \
+     configure-deb-control \
```

Comment:
These script files should also have the ${PACKAGE_BASE_NAME} prefix.

Reply:
removed this file to address deb generation as a separate PR